lib/DDG/Spice/Statista.pm

```diff
@@ -0,0 +1,28 @@
+package DDG::Spice::Statista;
+# ABSTRACT: Returns statistics from statista.com.
+
+use strict;
+use DDG::Spice;
+
+primary_example_queries "statistics about soccer";
+description "Shows example statistics";
+name "Statista";
+category "facts";
+
+my $api_key = '5db81c2a174170dac75620a16f553dd9';
```

Comment:
Please send me an API key to moollaza@duckduckgo.com, you can add a testing key to `duckpan` using:

```
duckpan env set DDG_SPICE_STATISTA_APIKEY <key>
```